### DISCUSSION TITLE
Problem in creating index using Meilisearch SDK 0.15.0

### DISCUSSION THREAD

**Discussion by miguelisidoro** [meilisearch/product#752] [Feedback & Feature Proposal]
Hello,

We are using https://www.nuget.org/packages/MeiliSearch/0.15.0 to interact with Meilisearch index when indexing content from our solution.

Our solution backend is implemented in C# (.NET 8).
The class we are using to hold the object that is as follows.

public class SearchItem
{
    [DataMember(Name = "contentItemId")]
    public string ContentItemId { get; set; }

    [DataMember(Name = "title")]
    public string Title { get; set; }

    [DataMember(Name = "text")]
    public string Text { get; set; }

    [DataMember(Name = "channelId")]
    public string[] ChannelId { get; set; }

    [DataMember(Name = "contentType")]
    public string ContentType { get; set; }

    [DataMember(Name = "fileExtension")]
    public string FileExtension { get; set; }

    [DataMember(Name = "targetAudience")]
    public string[] TargetAudience { get; set; }

    [DataMember(Name = "tags")]
    public string[] Tags { get; set; }

    [DataMember(Name = "publishingTimestamp")]
    public long PublishingTimestamp { get; set; }

    [DataMember(Name = "tenant")]
    public string Tenant { get; set; }

    [DataMember(Name = "_vectors", IsRequired = true)]
    public SearchItemVector Vectors { get; set; }
}

When using this class,  the index is not generated, although no errors are thrown in our solution.

![image](https://github.com/meilisearch/product/assets/5429152/78798748-3109-41f6-af9d-7b61ed556bed)

If we change the class to:

public class SearchItem
{
    [DataMember(Name = "contentItemId")]
    public string ContentItemId { get; set; }

    [DataMember(Name = "title")]
    public string Title { get; set; }

    [DataMember(Name = "text")]
    public string Text { get; set; }

    [DataMember(Name = "channelId")]
    public string[] ChannelId { get; set; }

    [DataMember(Name = "contentType")]
    public string ContentType { get; set; }

    [DataMember(Name = "fileExtension")]
    public string FileExtension { get; set; }

    [DataMember(Name = "targetAudience")]
    public string[] TargetAudience { get; set; }

    [DataMember(Name = "tags")]
    public string[] Tags { get; set; }

    [DataMember(Name = "publishingTimestamp")]
    public long PublishingTimestamp { get; set; }

    [DataMember(Name = "tenant")]
    public string Tenant { get; set; }

    [DataMember(Name = "_vectors", IsRequired = true)]
    public SearchItemVector _vectors { get; set; } //property changed from Vectors to _vectors
}

The index is correctly created with a _vectors field as I want.

![image](https://github.com/meilisearch/product/assets/5429152/d6a78cda-158e-45aa-a4c1-401c8052e5e4)

Why is the first version of the class not creating any documents in the index?
I don't want to have a property class named _vectors in my C# class.

Thanks

**Comment by curquiza** [comment]
Hello

Sorry it's not clear yet, we plan to make it clearer in a near future, but we don't handle support here. To get quickly help, we recommend to ask support of the community [here, on our Discord](https://discord.meilisearch.com/)

Thank you very much for using Meilisearch

**Reply by miguelisidoro** [reply]
Hello,

Isn't the SDK yours? Or was it developed by someone else?
If it is developed by you, you should handle support.

The discord doesn't work well. I ask questions, nobody answers. Someone must support this SDK, I believe that this should be your responsibility of handling support if it was you who developed it.

Thanks

Thanks
